resource/secret/create.go

```diff
@@ -0,0 +1,60 @@
+package secret
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/giantswarm/microerror"
+	v1 "k8s.io/api/core/v1"
```

Comment:
```suggestion
	corev1 "k8s.io/api/core/v1"
```

This needs updating in multiple places.